### DISCUSSION TITLE
Welcome to ovoid-API Discussions!

### DISCUSSION THREAD

**Discussion by namtxs** [namdevel/ovoid-API#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tenkuken** [comment]
Boleh sharing tools u/ reverse engineringnya kk?

**Reply by namtxs** [reply]
### jadx-gui [skylot/jadx](https://github.com/skylot/jadx)

![image](https://camo.githubusercontent.com/61fcd8ad7fe7f4d7792b85006f855ec26327b163580e48d86f0507790a4ef90c/68747470733a2f2f692e696d6775722e636f6d2f6839313749425a2e706e67)

**Comment by ryantedri** [comment]
get auth token kok bad request terus ya bg?
